azure-pipelines.yml

```
@@ -27,3 +27,5 @@ jobs:
     displayName: Run clang-format and clang-tidy
   - script: make -C src check
     displayName: Test C++ executables
+  - script: make -C data check python_executable=python m=0 prune_branches=0 prune_bridges=0
```

```suggestion
  - script: make -C data test python_executable=python m=0 prune_branches=0 prune_bridges=0
```
Just a suggestion to make it more clear this is a test